.semaphore/third-party-notices.yml

```diff
@@ -0,0 +1,40 @@
+version: v1.0
+name: third-party-notices
+agent:
+  machine:
+    type: s1-prod-ubuntu24-04-amd64-1
+
+auto_cancel:
+  running:
+    when: "branch != 'main'"
+
+execution_time_limit:
+  hours: 1
+
+queue:
+  - when: "branch != 'main'"
+    processing: parallel
+
+global_job_config:
+  env_vars:
+    - name: NODE_ENV
+      value: "production"
+
+blocks:
+  - name: "Update third party notices PR"
+    run:
+      when: "branch =~ '.*'"
+    task:
+      prologue:
+        commands:
+          - checkout
+          - sem-version java 21
+          - . vault-setup
```

Comment:
I believe these two lines are not needed. (I think they weren't needed for ide-sidecar either but that's fine).

```suggestion
```